mysql-test/t/percona_bug_ps260.test

```diff
@@ -0,0 +1,226 @@
+--source include/have_innodb.inc
```

Comment:
Sorry for missing this earlier: Since the crash is inside of InnoDB, it should go to the InnoDB suite. No need for Jenkins re-test and another review round, just move, test locally, and merge.